rpc/js/src/dial.ts

```diff
@@ -183,17 +224,19 @@ export async function dialWebRTC(signalingAddress: string, host: string, opts?:
 	if (opts) {
 		optsCopy = { ...opts } as DialOptions;
 
-		optsCopy.authEntity = opts?.webrtcOptions?.signalingAuthEntity
-		if (!optsCopy.authEntity) {
-			if (optsCopy.externalAuthAddress) {
-				optsCopy.authEntity = opts.externalAuthAddress?.replace(/^(.*:\/\/)/, '');
-			} else {
-				optsCopy.authEntity = signalingAddress.replace(/^(.*:\/\/)/, '');
+		if (!opts.accessToken) {
+			optsCopy.authEntity = opts?.webrtcOptions?.signalingAuthEntity
+			if (!optsCopy.authEntity) {
+				if (optsCopy.externalAuthAddress) {
+					optsCopy.authEntity = opts.externalAuthAddress?.replace(/^(.*:\/\/)/, '');
+				} else {
+					optsCopy.authEntity = signalingAddress.replace(/^(.*:\/\/)/, '');
+				}
 			}
+			optsCopy.credentials = opts?.webrtcOptions?.signalingCredentials
```

Comment:
this is where we'd pass through a signaling access token, if set, for the sake of dialing the signaler.